langwatch/src/components/simulations/SimulationCard.tsx

```
@@ -0,0 +1,71 @@
+            <Text fontWeight="bold">{title}</Text>
+            <Badge colorPalette={statusColor} size="sm" mt="1">
+              {status}
+            </Badge>
+          </VStack>
+          <Box
+            as="button"
+            aria-label="Expand"
+            opacity={0.7}
+            _hover={{ opacity: 1 }}
+            onClick={onExpandToggle}
+            cursor={!!onExpandToggle ? "pointer" : "auto"}
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)